src/libraries/System.Private.CoreLib/src/System/IO/FileStreamStrategy.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Win32.SafeHandles;
+
+namespace System.IO
+{
+    internal abstract class FileStreamStrategy : Stream
```

Comment:
As far as I know, it would not help from perf perspective. We would also lose some features that `Steam` is offering like `CopyToAsync` for `UnixFileStreamStrategy`.